polaris/ocean/tasks/merry_go_round/__init__.py

```diff
@@ -0,0 +1,46 @@
+from polaris import Step, Task
+from polaris.config import PolarisConfigParser
+from polaris.ocean.resolution import resolution_to_subdir
+from polaris.ocean.tasks.merry_go_round.default import Default
+from polaris.ocean.tasks.merry_go_round.init import Init
+
+
+def add_merry_go_round_tasks(component):
+    resolution = 5
```

Comment:
```suggestion
    resolution = 5.  # Resolution provided in [units]
```
Replace `[units]` with `m` or `km`